Gopkg.lock

```diff
@@ -617,7 +617,7 @@
     "transport/http/internal/propagation",
   ]
   pruneopts = "NUT"
-  revision = "3f6e8463aa1d824abe11b439d178c02220079da5"
+  revision = "9e7f2a839bfefc34110ebbc321850d5a7ad1ebd4"
```

Comment:
it would be nice to switch off `master` in `Gopkg.toml` for this reference, and onto a tagged release.  Does `v0.3.2` work here?  If so, let's update the `Gopkg.toml` like:
```
[[constraint]]
  name = "google.golang.org/api"
  version = "~v0.3.2"
```

Reply:
For sure, makes sense. `v0.3.2` is recent enough to have the `storageLocations` field.